coreum/assetlist.json

```diff
@@ -93,6 +93,178 @@
+            "chain_name": "xrpl",
+            "base_denom": "xrpl11278ecf9e"
+          },
+          "provider": "Coreum"
+        }
+      ],
+      "coingecko_id": "sologenic",
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "xrpl",
+            "base_denom": "xrpl11278ecf9e"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/solo.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/solo.svg",
+          "theme": {
+            "primary_color_hex": "#ffffff"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/solo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/solo.svg"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "RLUSD bridged from XRPL",
+      "denom_units": [
+        {
+          "denom": "xrpl570c00a604-core1zhs909jp9yktml6qqx9f0ptcq2xnhhj99cja03j3lfcsp2pgm86studdrz",
+          "exponent": 0,
+          "aliases": [
+            "xrpl570c00a604"
+          ]
+        },
+        {
+          "denom": "rlusd",
+          "exponent": 15
+        }
+      ],
+      "base": "xrpl570c00a604-core1zhs909jp9yktml6qqx9f0ptcq2xnhhj99cja03j3lfcsp2pgm86studdrz",
+      "name": "RLUSD",
+      "display": "rlusd",
+      "symbol": "RLUSD",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "xrpl",
+            "base_denom": "xrpl570c00a604"
+          },
+          "provider": "Coreum"
+        }
+      ],
+      "coingecko_id": "ripple-usd",
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "xrpl",
+            "base_denom": "xrpl570c00a604"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/rlusd.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/rlusd.svg",
+          "theme": {
+            "primary_color_hex": "#1b90f7"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/rlusd.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/rlusd.svg"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "SARA is the utility native token of the Pulsara ecosystem. The burn or mint of the $SARA token will be decided by the community through proposals and voting.",
+      "denom_units": [
+        {
+          "denom": "usara-core1r9gc0rnxnzpq33u82f44aufgdwvyxv4wyepyck98m9v2pxua6naqr8h03z",
+          "exponent": 6
+        }
+      ],
+      "base": "usara",
```

Comment:
This denom is not defined in the denom_units.  It's likely either "usara-core1r9gc0rnxnzpq33u82f44aufgdwvyxv4wyepyck98m9v2pxua6naqr8h03z" or just "core1r9gc0rnxnzpq33u82f44aufgdwvyxv4wyepyck98m9v2pxua6naqr8h03z"

Reply:
Fixed